src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt

```diff
@@ -110,9 +110,9 @@ class WLSService(
                         "WLS DB has $itemsInStockQuantity stocked, and storage system tried to pick $pickedItemsQuantity"
                 }
             }
-            val quantity = Math.clamp(itemsInStockQuantity.minus(pickedItemsQuantity), 0.0, Double.MAX_VALUE)
+            val quantity = Math.clamp(itemsInStockQuantity.minus(pickedItemsQuantity).toLong(), 0, Int.MAX_VALUE)
```

Comment:
Shouldn't that be toInt instead of toLong?

Reply:
It is already an int, but the clamp function surprisingly does not take in an int.